models/convert/paths_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/onflow/flow-go/ledger"
+)
+
+func TestPathsToBytes(t *testing.T) {
+	path1b := []byte("aac513eb1a0457700ac3fa8d292513e1")
+	path2b := []byte("1454ae2420513f79f6a5e8396d033369")
+	path3b := []byte("e91a3eb997752b78bab0bc31e30b1e30")
+
+	path1, _ := ledger.ToPath(path1b)
+	path2, _ := ledger.ToPath(path2b)
+	path3, _ := ledger.ToPath(path3b)
```

Comment:
Are we sure they work? It looks to me like the paths are too short and would fail, and the test case passes anyway, because everything comes out empty.

Overall, I think it's a bad idea to rely on external code to test. We should just initialize a couple of byte slices _and_ the ledger paths explicitly.

Reply:
They work though:

```go
	t.Log(path1) // 01100001...00110001
	t.Log(path2) // 00110001...00111001
	t.Log(path3) // 01100101...00110000
```

But you're right, by using `ledger.ToPath` we might not even realize if they change the underlying type as the test would still pass (and the code would still work). Maybe it makes sense to ensure that our test would break if they change the way paths are, so that we can double-check that it doesn't have a hidden impact on our side.